apis/PreferencesEndpoint.es6.js

```diff
@@ -0,0 +1,26 @@
+import { runQuery, runForm } from '../apiBase/APIRequestUtils';
+
+import Preferences from '../models2/Preferences';
+
+const prefsUrl = '/api/v1/me/prefs';
+
+// We opt-out of using the automatic parsing from `runQuery` and `runForm`,
+// because the preferences object doesn't really make sense in the normalized
+// response model.
+
+export default {
+  get: async (apiOptions) => {
+    const responseBody = await runQuery(apiOptions, 'get', prefsUrl, {}, {});
```

Comment:
:no_bell: can we assume something called `runQuery` is always going to be a GET?